src/main/java/edu/cornell/kfs/module/purap/batch/service/impl/JaggaerGenerateSupplierXmlServiceImpl.java

```diff
@@ -329,7 +333,7 @@ protected List<SupplierSyncMessage> buildSupplierSyncMessageList(List<Supplier>
     private Header buildHeader() {
         Header header = new Header();
         header.setMessageId(UUID.randomUUID().toString());
-        header.setTimestamp(dateTimeService.getLocalDateTimeNow().format(DATE_TIME_ZONE_UTC_FORMATTER_yyyy_MM_dd_T_HH_mm_ss_SSS_Z));
+        header.setTimestamp(LocalDateTime.now(utcClock).format(DATE_TIME_ZONE_UTC_FORMATTER_yyyy_MM_dd_T_HH_mm_ss_SSS_Z));
```

Comment:
is this change needed? The old code was header.setTimestamp(DATE_FORMATTER_FOR_HEADER_DATE.print(dateTimeService.getCurrentDate().getTime()));

Is this going to change the timestamp in the header when it shouldn't?

Reply:
I think we should look at old files in DEV/TEST to see what the header date was. Unfortunately I am getting a 502 when trying to get in either DEV or TEST this morning.